### ISSUE TITLE
aider freezers while scanning repo. 

### ISSUE THREAD

**Issue by wsalem117** [Aider-AI/aider#2505]
This is a new instance for me here, where whenever i try to load aider it will just freeze while scanning repo and it will just not finish. 

I tested in multiple VSCODE projects and still the same behavior. and I tried different models. Anthropic sonnet 3.5 and gpt4o and getting the same. 

I deleted all aider folders and files in the project and still go the same 

![image](https://github.com/user-attachments/assets/3f045ef7-34a6-4ed2-adcb-6c7efec75d6e)

Also when I try to kill the application it will not just terminate I have to delete the complete terminal nad re-start 

After the above screen shot I killed my terminal I restarted again and lunched iwht 40 and stuck on this for the past 5 minutes 

![image](https://github.com/user-attachments/assets/ab49ad3b-feb9-45b8-9627-df0cb23ebdec)

Now I deleted all my aider folders again within the project and re-tried again 

![image](https://github.com/user-attachments/assets/12fc6bf8-bca6-4c32-9d0c-4ea468d0bbb7)

Now again during scanning repo gets stuck at 75% and does not move again. 

then I uninstalled and did a fresh install 

then when I tried to run again 

![image](https://github.com/user-attachments/assets/f783b86a-72c6-4a88-b450-955f3d39ee8b)

Then when I deleted all cache and tried to re-run again same issue here 

![image](https://github.com/user-attachments/assets/3c7d320c-46f9-4a1a-a21e-34face060861)

**Comment by paul-gauthier** [comment]
Thanks for trying aider and filing this issue.

Try rubbing with --verbose and see if it's getting stuck on a particular file.

**Comment by wsalem117** [comment]
> Thanks for trying aider and filing this issue.
> 
> Try rubbing with --verbose and see if it's getting stuck on a particular file.

![image](https://github.com/user-attachments/assets/5c619ae3-934e-45dc-b285-d933a6a3e4a3)

I started getting this issue again. and when I ran verbose I see it always gets stuck at this file. The file is just a simple table nothing complex and the file has no business logic or code errors and follows basic code limiting structure. 

to bypass this, I have the comment or delete the code, and the only the repo map works otherwise it just gets stuck there and does not exit this error
